Makefile.am

```diff
@@ -158,14 +158,66 @@ base_files = \
 bin_PROGRAMS += csd
 
 csd_SOURCES = \
-	${base_files} \
+	${BASE_FILES} \
```

Comment:
BASE_FILES will be built for each target binary, i.e. you will have a different .o for each binary you are building.